src/server/sql/unit/get_by_unit_index_meter.sql

```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+SELECT id
+FROM units 
+WHERE unit_type = 'meter' AND identifier = ${unitIndex};
```

Comment:
I've tried to use `unit_type.meter` but it failed. [The Postgres documentation about enum](https://www.postgresql.org/docs/9.1/datatype-enum.html) just does the same as the code above.

However, for safety, I think we can replace `'meter'` with `'meter'::unit_type`. This will cast `meter` to `unit_type` and will throw an error if `'meter'` is not a value of the `unit_type` enum. What do you think about this?

Reply:
That sounds fine. We don't expect the OED code to ever pass anything else but checks are always good.

Reply:
I've changed other places as well so I'll mark them as resolved.